### PR TITLE
Issue 931 - Fix BCF file upload 

### LINKED ISSUE #931: Browser doesn't pick up file on BCF import
1. go to Issues menu
2. Import BCF
3. select file and hit Open
4. often nothing happens

### PR DESCRIPTION
**Pull request by JamesMilnerUK**
This fixes #931 

#### Description
File changes were not being detected for BCF imports making them flaky

#### Test cases
Uploading a BCF file
Uploading the same BCF file twice in a row

### CODE REVIEW

**Comment by carmenfan** [comment]
@JamesMilnerUK  I changed it to merge into staging. Don't think this is worthy of a hot fix

**Comment by JamesMilnerUK** [comment]
@carmenfan cool beans

**Comment by Charence** [comment]
BCF import error causes backend to crash

screenshot 2018-04-30 15 57 38
screenshot 2018-04-30 15 57 38 2

File: frontend/components/issues/js/issues.component.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ class IssuesController implements ng.IController {
 		"ClientConfigService",
 		"AnalyticService",
 		"DialogService",
-		"ViewerService"
+		"ViewerService",
```
Comment (Charence) [inline frontend/components/issues/js/issues.component.ts]: build fails ts-lint here

/home/travis/build/3drepo/3drepo.io/frontend/components/issues/js/issues.component.ts:35:18
ERROR: 35:18  trailing-comma              Unnecessary trailing comma
ERROR: 84:24  trailing-comma              Unnecessary trailing comma

File: frontend/components/issues/js/issues.component.ts [hunk 2]
```diff
@@ -81,7 +81,7 @@ class IssuesController implements ng.IController {
 		private ClientConfigService,
 		private AnalyticService,
 		private DialogService,
-		private ViewerService
+		private ViewerService,
```
Comment (Charence) [inline frontend/components/issues/js/issues.component.ts]: build fails ts-lint here

/home/travis/build/3drepo/3drepo.io/frontend/components/issues/js/issues.component.ts:35:18
ERROR: 35:18  trailing-comma              Unnecessary trailing comma
ERROR: 84:24  trailing-comma              Unnecessary trailing comma

**Pull request merged**